docs/cloud/satellites/self-hosted.md

```diff
@@ -0,0 +1,215 @@
+# Self-Hosted Satellites
+
+Self-hosted satellites _\*Beta\*_ are a good alternative to Earthly Cloud’s [managed satellites](../satellites.md) when CI/CD is required to run in your own environment. Self-hosted satellites provide most of the benefits of Earthly Cloud while ensuring that build-related data never leaves your network.
+
+In addition, Self-Hosted Satellites have the following features:
+* Lightning-fast caching
+* Cloud-enabled control plane
+* Encrypted connections by default using mTLS
+* Ready to run builds with just a single command
+
+On the other hand, they may have the following drawbacks when compared to managed satellites offered in Earthly Cloud:
```

Comment:
@thtmnisamnstr - thoughts on what to call the regular satellites? I think we use the term "Cloud" on our pricing. Do we then call these "Cloud Satellites" instead of managed satellites?

Reply:
That makes sense. Cloud Satellites, Self-Hosted Satellites, and Dedicated Satellites.